intel_extension_for_transformers/neural_chat/server/config/config.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from dotenv import load_dotenv
+
+from functools import lru_cache
+
+from pydantic import BaseSettings
+
+from intel_extension_for_transformers.utils import logger
+
+load_dotenv()
+
+
+class Settings(BaseSettings):
+    mysql_user: str = os.environ.get("MYSQL_USER", "root")
```

Comment:
better to organize those settings in a more structured way e.g., database related, auth related (base auth, child auth ...)